### PR TITLE
Terse route fixes

### PR DESCRIPTION
**Pull request by ddeaguiar**
Aligns the Terse route documentation in the `Routing Quick Reference` page with Pedestal's implementation.

Also updates the site copyright and README.

This PR addresses #91.

### CODE REVIEW

**Comment by ddeaguiar** [comment]
@mtnygard not sure why the ci build is trying to push this branch to prod but that's why it failed.

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Retrieve Pedestal and switch to a publicly-available version:
 
     git clone https://github.com/pedestal/pedestal.git pedestal
     cd pedestal
-    git checkout 0.5.1
+    git checkout 0.5.3
```
Comment (solussd) [inline README.md]: hasn't 0.5.4 been cut?

File: content/reference/routing-quick-reference.adoc [hunk 2]
```diff
@@ -206,7 +210,14 @@ The allowed keys in a verb map are:
 - :delete
 - :any
 
-The value of each key is either a handler function or a list of interceptors.
+The value of each key is either:
```
Comment (solussd) [inline content/reference/routing-quick-reference.adoc]: consider "is one of"

Comment (ddeaguiar) [reply content/reference/routing-quick-reference.adoc]: sure.

File: README.md (hunk 1 above)
Comment (ddeaguiar) [reply README.md]: Yes, this PR predates that. I can update.

**Pull request merged**